cl/expr.go

```diff
@@ -687,6 +688,14 @@ func compileSliceLit(ctx *blockCtx, v *ast.SliceLit) {
 	ctx.cb.SliceLit(nil, n)
 }
 
+func compileSliceStep(ctx *blockCtx, v *ast.SliceStep) {
+	compileExpr(ctx, v.StartExpr)
+	compileExpr(ctx, v.EndExpr)
+	compileExpr(ctx, v.StepExpr)
+
+	ctx.cb.SliceStep(nil)
```

Comment:
You can implements this feature by https://github.com/goplus/gop#for-range-of-udt2

Reply:
We can suppose that Low, High, Step is all an expression whose result is `int`.